### ISSUE TITLE
`mocha is not defined` error when MOCHA_GREP is used

### ISSUE THREAD

**Issue by Firfi** [DispatchMe/meteor-mocha#26]
I'm getting `mocha is not defined` error *only* when MOCHA_GREP with any string (i.e. MOCHA_GREP=billing) is used. 

Full stack trace: 

```
I20170424-00:05:57.168(7)? --------------------------------
I20170424-00:05:57.169(7)? ----- RUNNING SERVER TESTS -----
I20170424-00:05:57.169(7)? --------------------------------
I20170424-00:05:57.172(7)?
W20170424-00:05:57.192(7)? (STDERR) /Users/firfi/.meteor/packages/meteor-tool/.1.4.3_2.joqega++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
W20170424-00:05:57.192(7)? (STDERR) 						throw(ex);
W20170424-00:05:57.193(7)? (STDERR) 						^
W20170424-00:05:57.195(7)? (STDERR)
W20170424-00:05:57.196(7)? (STDERR) ReferenceError: mocha is not defined
W20170424-00:05:57.197(7)? (STDERR)     at serverTests (packages/dispatch:mocha/server.js:85:13)
W20170424-00:05:57.200(7)? (STDERR)     at Object.start (packages/dispatch:mocha/server.js:140:5)
W20170424-00:05:57.201(7)? (STDERR)     at packages/meteor.js:1044:27
W20170424-00:05:57.203(7)? (STDERR)     at Function.time (/private/var/folders/jl/7wz_34m90s96vfzc0_jjb5n00000gn/T/meteor-test-runthih7s/.meteor/local/build/programs/server/profile.js:309:28)
W20170424-00:05:57.203(7)? (STDERR)     at /private/var/folders/jl/7wz_34m90s96vfzc0_jjb5n00000gn/T/meteor-test-runthih7s/.meteor/local/build/programs/server/boot.js:312:13
W20170424-00:05:57.206(7)? (STDERR)     at /private/var/folders/jl/7wz_34m90s96vfzc0_jjb5n00000gn/T/meteor-test-runthih7s/.meteor/local/build/programs/server/boot.js:353:5
W20170424-00:05:57.208(7)? (STDERR)     at Function.run (/private/var/folders/jl/7wz_34m90s96vfzc0_jjb5n00000gn/T/meteor-test-runthih7s/.meteor/local/build/programs/server/profile.js:510:12)
W20170424-00:05:57.209(7)? (STDERR)     at /private/var/folders/jl/7wz_34m90s96vfzc0_jjb5n00000gn/T/meteor-test-runthih7s/.meteor/local/build/programs/server/boot.js:351:11
```

Meteor 1.4.x

**Comment by Firfi** [comment]
It's possible that the issue is somewhere in `import { mocha } from 'meteor/practicalmeteor:mocha-core';`

**Comment by aldeed** [comment]
@Firfi Can you post the whole command you're using to run the tests? I am not able to reproduce this. You are using latest v0.3.0 of this driver pkg?

**Comment by Firfi** [comment]
@aldeed command is `MOCHA_GREP=Billing meteor test --once --driver-package dispatch:mocha`

in versions: `dispatch:mocha@0.3.0`

**Comment by aldeed** [comment]
@Firfi This should be fixed in 0.4.1 now
